mainsite/pages/whitepaper.js

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react'
+import styled from '@emotion/styled'
+import { flex } from 'emotion-styled-utils'
+
+import { content, images } from '../docs/whitepaper'
+import { withApollo } from '../src/frontend/hoc'
+import Layout from '../src/frontend/components/Layout'
+import Markdown from '../src/frontend/components/Markdown'
+import FundUs from '../src/frontend/components/FundUs'
+import Seo from '../src/frontend/components/Seo'
+import ContentWrapper from '../src/frontend/components/ContentWrapper'
```

Comment:
We should try and keep these warnings down to a minimum

Reply:
At some point we should go through and clean up all lint warnings.